test/mocks/fake.go

```diff
@@ -20,7 +20,7 @@ type Fake struct {
 	t    *testing.T       // testing instance
 	ns   string           // namespace
 	S    *runtime.Scheme  // runtime client scheme
-	objs []runtime.Object // all fake objects
+//	objs []runtime.Object // all fake objects
```

Comment:
Remove comment here as well. Code will be cleaner.

Reply:
solved